<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-KLPCFEN" failures="0" tests="15" name="com.wolvesres.quanghn.voucher.TestValidVoucher" time="0.006" errors="2" timestamp="2022-04-15T13:46:07 ICT" skipped="0">
  <testcase classname="com.wolvesres.quanghn.voucher.TestValidVoucher" name="testValidVoucherSuccess" time="0.001"/>
  <system-out/>
  <testcase classname="com.wolvesres.quanghn.voucher.TestValidVoucher" name="testValidVoucherFail" time="0.000"/>
  <system-out/>
  <testcase classname="com.wolvesres.quanghn.voucher.TestValidVoucher" name="testValidVoucherSuccess" time="0.000"/>
  <system-out/>
  <testcase classname="com.wolvesres.quanghn.voucher.TestValidVoucher" name="testValidVoucherChonNgayFail" time="0.000"/>
  <system-out/>
  <testcase classname="com.wolvesres.quanghn.voucher.TestValidVoucher" name="testValidVoucherChonNgayFail" time="0.000"/>
  <system-out/>
  <testcase classname="com.wolvesres.quanghn.voucher.TestValidVoucher" name="testValidVoucherSuccess" time="0.001"/>
  <system-out/>
  <testcase classname="com.wolvesres.quanghn.voucher.TestValidVoucher" name="testValidVoucherFail" time="0.000">
    <error message="For input string: &quot;-56,89&quot;" type="java.lang.NumberFormatException">
      <![CDATA[java.lang.NumberFormatException: For input string: "-56,89"
at java.base/jdk.internal.math.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2054)
at java.base/jdk.internal.math.FloatingDecimal.parseFloat(FloatingDecimal.java:122)
at java.base/java.lang.Float.parseFloat(Float.java:476)
at com.wolvesres.quanghn.voucher.TestValidVoucher.testValidVoucherFail(TestValidVoucher.java:60)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- testValidVoucherFail -->
  <system-out/>
  <testcase classname="com.wolvesres.quanghn.voucher.TestValidVoucher" name="testValidVoucherFail" time="0.000">
    <error message="For input string: &quot;-767,8&quot;" type="java.lang.NumberFormatException">
      <![CDATA[java.lang.NumberFormatException: For input string: "-767,8"
at java.base/jdk.internal.math.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2054)
at java.base/jdk.internal.math.FloatingDecimal.parseFloat(FloatingDecimal.java:122)
at java.base/java.lang.Float.parseFloat(Float.java:476)
at com.wolvesres.quanghn.voucher.TestValidVoucher.testValidVoucherFail(TestValidVoucher.java:60)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- testValidVoucherFail -->
  <system-out/>
  <testcase classname="com.wolvesres.quanghn.voucher.TestValidVoucher" name="testValidVoucherChonNgayFail" time="0.000"/>
  <system-out/>
  <testcase classname="com.wolvesres.quanghn.voucher.TestValidVoucher" name="testValidVoucherChonNgayFail" time="0.001"/>
  <system-out/>
  <testcase classname="com.wolvesres.quanghn.voucher.TestValidVoucher" name="testValidVoucherFail" time="0.001"/>
  <system-out/>
  <testcase classname="com.wolvesres.quanghn.voucher.TestValidVoucher" name="testValidVoucherChonNgayFail" time="0.001"/>
  <system-out/>
  <testcase classname="com.wolvesres.quanghn.voucher.TestValidVoucher" name="testValidVoucherSuccess" time="0.001"/>
  <system-out/>
  <testcase classname="com.wolvesres.quanghn.voucher.TestValidVoucher" name="testValidVoucherFail" time="0.000"/>
  <system-out/>
  <testcase classname="com.wolvesres.quanghn.voucher.TestValidVoucher" name="testValidVoucherSuccess" time="0.000"/>
  <system-out/>
</testsuite> <!-- com.wolvesres.quanghn.voucher.TestValidVoucher -->
